homeassistant/components/ecobee/climate.py

```diff
@@ -84,13 +92,32 @@
     PRESET_HOLD_INDEFINITE: "indefinite",
 }
 
-SERVICE_SET_FAN_MIN_ON_TIME = "set_fan_min_on_time"
+SERVICE_CREATE_VACATION = "create_vacation"
+SERVICE_DELETE_VACATION = "delete_vacation"
 SERVICE_RESUME_PROGRAM = "resume_program"
+SERVICE_SET_FAN_MIN_ON_TIME = "set_fan_min_on_time"
 
-SET_FAN_MIN_ON_TIME_SCHEMA = vol.Schema(
+CREATE_VACATION_SCHEMA = vol.Schema(
     {
-        vol.Optional(ATTR_ENTITY_ID): cv.entity_ids,
-        vol.Required(ATTR_FAN_MIN_ON_TIME): vol.Coerce(int),
+        vol.Required(ATTR_ENTITY_ID): cv.entity_id,
+        vol.Required(ATTR_VACATION_NAME): cv.string,
+        vol.Required(ATTR_COOL_TEMP): vol.Any(cv.positive_int, float),
```

Comment:
Consider using `vol.Coerce(float)` and a range.

Reply:
Coercing to float is ok - the ecobee API doesn't specify a range for acceptable temperature values as this appears to be dependent on user preferences and the firmware of the particular thermostat. If a user tries to set a value out of the allowable range, I believe that the API will automatically set the value to the max/min allowable by the thermostat.